packages/e2e-selenium/util/create-new-post.js

```diff
@@ -0,0 +1,7 @@
+const createNewPost = async () => {
+	await driver.get( 'http://localhost:8888/wp-admin/post-new.php' );
```

Comment:
It might be better to use `localhost:8889` instead, as it's specifically set up for testing. `8888` being the environment used for development, tests might be affected by currently active plugins or other local changes.

Reply:
Fixed.